src/Backoffice/Tasks/App/Requests/UpsertTaskRequest.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Tasks\App\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class UpsertTaskRequest extends FormRequest
+{
+    public function rules(): array
+    {
+        return [
+            'id' => [ 'integer', 'exists:tasks,id'],
+            'title' => 'required',
+            'description' => 'required',
+            'employee_id' => ['required', 'exists:employees,id'],
+            'status'=> ['required'],
```

Comment:
```suggestion
            'id' => ['integer', 'nullable', 'exists:tasks,id'],
            'title' => ['required', 'max:255'],
            'description' => ['required', 'max:5000'],
            'employee_id' => ['required', 'exists:employees,id'],
            'status'=> ['required'],
```
I shouldn't be able to provide whatever value I wish for `status`. We usually use enums for this type of case. For reference:
```
enum Status: string
{
    case Active = 'active';
    case Draft = 'draft';
}
```
And in your FormRequest:
```
'status'=> ['required', Rule::enum(Status::class)],
```

Reply:
Fixed and also created enum for status